person.rb

```diff
@@ -0,0 +1,16 @@
+class Person
+
+  attr_reader :age, :salary, :phase, :target_raise
```

Comment:
Why do all people have salary and target_raise?  The `Student` class probably doesn't need those things.